examples/v1beta1/100-cpu-limit.yaml

```
@@ -0,0 +1,28 @@
+# This example NodePool limits the amount of compute
+# NodePool by Karpenter to 100 CPU cores
+apiVersion: karpenter.sh/v1beta1
+kind: NodePool
+metadata:
+  name: default
```

in-line with @sftim's recommendation below, I think we could change this 
```suggestion
  name: limitcpu100
  annotations:
    kubernetes.io/description: "No more than 100 cpu provisioned"```